seqr/utils/logging_utils.py

```diff
@@ -73,14 +73,16 @@ def log_model_update(logger, model, user, update_type, update_fields=None):
     logger.info('{} {} {}'.format(update_type, db_entity, entity_id), user, db_update=db_update)
 
 
-def log_model_bulk_update(logger, models, user, update_type, update_fields=None):
+def log_model_bulk_update(logger, models, user, update_type, update_fields=None, parent=None):
     if not models:
         return []
     db_entity = type(models[0]).__name__
-    entity_ids = [o.guid for o in models]
+    entity_ids = [o.guid if hasattr(o, 'guid') else o.id for o in models]
     db_update = {
         'dbEntity': db_entity, 'entityIds': entity_ids, 'updateType': 'bulk_{}'.format(update_type),
     }
+    if parent:
+        db_update['parentEntityIds'] = list({getattr(model, parent).guid for model in models})
```

Comment:
This does not belong in a generic helper method. For the one use case where you care about this, just write a regular looger.info message that says something like `priortization updated for 47 samples` or something like that, we don't need it to be a structured audit data.

Reply:
I mimic the existing logging of RNA-seq data and log the `parent` models. What information should be included in the `db_update`?

Reply:
discussed offline, but this function should not change at all. You have 2 options:
1) Make a different logging helper for bulk updating non-guid models. It will not include any parent info or the `entityIds` list, probably add an `entityCount` instead
2) Don't treat the  data models as the thing that is being updated, log it as if the parents themselves are being updated. So instead of passing a list of  RNA or prioritization models here, pass the Individual or sample models directly, and pass the name of the set being updated in `update_fields` (i.e `update_fields=['phenotypeprioritization]`). In this case you would not put the logging in the bulk update helper function, you would call it explicitly afterwards

Reply:
I will do the option 1.